10-gis.Rmd

```diff
@@ -246,26 +246,38 @@ One of the algorithms on the above list, `"native:union"`, sounds promising.
 The next step is to find out what this algorithm does and how we can use it.
 This is the role of the `qgis_show_help()`, which returns a short summary of what the algorithm does, its arguments, and outputs.^[We can also extract some of information independently with `qgis_description()`, `qgis_arguments()`, and `qgis_outputs()`.]
 This makes it output rather long.
-Here, we only present the arguments`"native:union"` expects.
+The following command returns a data frame with each row representing an argument required by `"native:union"` and columns with the name, description, type, default value, available values, and acceptable values associated with each:
 
 ```{r 09-gis-6, eval=qgis}
-alg = "native:union"
-qgis_arguments(alg) |>
-  dplyr::mutate(acceptable_values = unlist(acceptable_values)) |>
```

Comment:
This line caused the following error: https://github.com/geocompx/geocompr/actions/runs/4647331273/jobs/8224278198#step:5:2000